### ISSUE TITLE
Suggestion: Attachment SyncLock start Off by Default

### ISSUE THREAD

**Issue by tiviness** [OpenCollarTeam/OpenCollar#292]
What version of OpenCollar are you using? 7.4 Beta 2

What behavior did you expect? When attaching cuffs, regardless of collar lock state, i expected cuffs to remain unlocked until manually locked.

What behavior did you see instead? When attaching cuffs, after settings loaded, they locked automatically, because SyncLock (Apps -- Attachments -- Settings -- Sync -- SyncLock) is on by default.

What steps does someone need to take to reproduce the problem? Make sure a 7.4 Beta 2 collar is attached and locked.  SyncLock will be on by default.  If one attaches cuffs, they will automatically lock.

i am hoping that, instead, it can be implemented as off by default.

**Comment by SilkieSabra** [comment]
This makes sense to me since more users will be surprised by the lock state and not knowing how to turn it off, than there will be users wanting to turn it on.  There are many occasions in ordinary role-play where cuffs wearers expect them to be unlocked until they are locked by someone; but more collars are always locked.  There will be a note in the manual about how to sync lock and unsync, either way.

**Comment by lillith-xue** [comment]
So clicking the Cuff and pressing unlock is no option?

**Comment by SilkieSabra** [comment]
can the cuffs be unlocked by the wearer if they are owned and Ownself is not checked?

**Comment by lillith-xue** [comment]
No they should behave same as the collar

**Comment by tiviness** [comment]
Unchecking lock (or attempting to) behaves as expected, per Lillith's comment when one is Owned and Ownself is not checked.

However, SyncLock was unchecked and Ankle Cuffs still locked upon loading.  i understand it's a low-level priority thing.  i only noticed because i had to wear cuffs for a posing while i had the collar on, and i had my collar locked.  Nothing high-priority or pixel life or death.  <3

**Comment by lillith-xue** [comment]
Ok if SyncLock was disabled and it locked by itself, it is a bug and needs to be fixed. Thank you for telling :-)

**Comment by tiviness** [comment]
Yeah i only just noticed that part when double checking roan's last question.  Glad to have looked deeper into it.  Have fun!  :-)

**Comment by zontreck** [comment]
Looking into this issue
